Reconstruction/RecCalorimeter/options/runEcalReconstruction.py

```diff
@@ -15,22 +15,40 @@
 
 #Configure tools for calo reconstruction
 from Configurables import CalibrateCaloHitsTool
-calibcells = CalibrateCaloHitsTool("CalibrateCaloHitsTool",invSamplingFraction="5.4")
+calibcells = CalibrateCaloHitsTool("CalibrateCaloHitsTool", invSamplingFraction="5.4")
 
 from Configurables import CreateCaloCells
 createcells = CreateCaloCells("CreateCaloCells",
                               calibTool=calibcells, doCellCalibration=True,
-                              addCellNoise=True,filterCellNoise=False,
-                              readoutName="ECalHitsPhiEta",numVolumesRemove=1,fieldNames=["system","ECAL_Cryo","bath","EM_barrel"],fieldValues=[5,1,1,1],
+                              addCellNoise=True, filterCellNoise=False,
+                              readoutName="ECalHitsPhiEta",
+                              fieldNames=["system","ECAL_Cryo","bath","EM_barrel"],
+                              fieldValues=[5,1,1,1],
                               OutputLevel=INFO)
 createcells.DataInputs.hits.Path="ECalHits"
 createcells.DataOutputs.cells.Path="caloCells"
 
 from Configurables import CreatePositionedHit
-positionhit = CreatePositionedHit("CreatePositionedHit", readoutName = "ECalHitsPhiEta",activeFieldName = "active_layer",activeVolumeName="LAr",numVolumesRemove="1")
+positionhit = CreatePositionedHit("CreatePositionedHit", readoutName = "ECalHitsPhiEta",activeFieldName = "active_layer",activeVolumeName="LAr_sensitive")
 positionhit.DataInputs.caloCells.Path="caloCells"
 positionhit.DataOutputs.caloPositionedHits.Path="caloCellsPositions"
 
+#Create calo clusters
+from Configurables import CreateCaloClustersSlidingWindow
+from GaudiKernel.PhysicalConstants import pi
+createclusters = CreateCaloClustersSlidingWindow("CreateCaloClusters",
+                                                 readoutName = "ECalHitsPhiEta",
+                                                 fieldNames = ["system","ECAL_Cryo","bath","EM_barrel"],
```

Comment:
"system" is the name of the top-most readout field. It corresponds to the detector ID (here: ECal).